src/Cases/Cases.tsx

```diff
@@ -52,7 +54,7 @@ class Cases extends Component {
 		return accu;
 	}, {});
 
-	render() {
+	public render() {
```

Comment:
Really happy to see the `public` access modifier. Can we actually mark all the other render functions (e.g. renderIntro) `private`? Only render is supposed to be exposed for React, right?

Reply:
Yes, all lifecycle methods should be `public`. The rest can be `private` or `static` 😃